cmd/server/assets/home.html

```diff
@@ -45,22 +48,29 @@ <h1>Issue diagnosis verification code</h1>
               Confirmed positive test result from an official testing source
             </small>
           </div>
+
+          {{if not $currentRealm.DisableLikelyReports}}
           <div class="form-check">
             <input class="form-check-input" type="radio" name="testType" id="testType2" value="likely" />
             <label class="form-check-label" for="testType2">Likely diagnosis</label>
             <small class="form-text text-muted">
               Clinical diagnosis without a test
             </small>
           </div>
+          {{end}}
+
+          {{if not $currentRealm.DisableNegativeReports}}
```

Comment:
this could be nested - it doesn't make sense to have negative without likely given the revision model.